github/interactions_repos.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// GetInteractions fetches the interaction restrictions for a repository.
+//
+// GitHub API docs: https://developer.github.com/v3/interactions/repos/#get-interaction-restrictions-for-a-repository
+func (s *InteractionsService) GetInteractions(ctx context.Context, owner string, repo string) (*Interaction, *Response, error) {
```

Comment:
Since this is the `InteractionsService`, what do you think about naming these functions `GetRestrictions`, `UpdateRestrictions`, and `RemoveRestrictions`?

Reply:
I agree with this. Will address.